src/utilities/hash.ts

```diff
@@ -0,0 +1,5 @@
+export type KeccakHash = string;
```

Comment:
This type is exported in its own file already

Reply:
Fixed